spec/features/users/profile_recovery_for_gpo_verified_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.feature 'Password recovery via personal key for a GPO-verified user' do
+  include IdvStepHelper
+
+  let(:user) { create(:user, :fully_registered) }
+  let(:new_password) { 'some really awesome new password' }
+
+  before do
+    allow(FeatureManagement).to receive(:reveal_gpo_code?).and_return(true)
+  end
+
+  scenario 'lets them reactivate their profile with their personal key', email: true, js: true do
+    complete_idv_steps_with_gpo_before_confirmation_step(user)
+    click_on 'Continue'
+
+    click_on t('account.index.verification.reactivate_button')
+    click_on t('idv.gpo.form.submit')
+
+    personal_key = scrape_personal_key
+    check t('forms.personal_key.required_checkbox')
+    click_continue
+
+    click_on 'Sign out'
```

Comment:
Can we update this to reference by `t('...')` key so that it's consistent with other specs and avoids desync issues with the underlying text?

Reply:
Done.